### PR TITLE
♻️Computation backend: clean comp_tasks DB (🗃️)

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- removes "submit" column in comp_tasks

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
:+1:

File: services/director-v2/src/simcore_service_director_v2/api/routes/computations.py [hunk 1]
```diff
@@ -554,26 +551,29 @@ async def stop_computation(
...
-            ),
-            url=TypeAdapter(AnyHttpUrl).validate_python(f"{request.url}"),
-            stop_url=None,
-            iteration=last_run.iteration if last_run else None,
-            result=None,
-            started=compute_pipeline_started_timestamp(pipeline_dag, tasks),
-            stopped=compute_pipeline_stopped_timestamp(pipeline_dag, tasks),
-            submitted=compute_pipeline_submitted_timestamp(pipeline_dag, tasks),
-        )
+        return await _create_computation_response()
```
Comment (GitHK) [inline services/director-v2/src/simcore_service_director_v2/api/routes/computations.py:576]: Q: out of curiosity why is this wrapped in a function call? could it not have worked without defining a new function?

**Review by matusdrobuliak66 — APPROVED** [review]
:100: Please do not forget DB migration icon in the name 🗃️

File: services/director-v2/src/simcore_service_director_v2/api/routes/computations.py (hunk 1 above)
Comment (sanderegg) [reply services/director-v2/src/simcore_service_director_v2/api/routes/computations.py:576]: that is a mistake. thanks for checking

Comment (sanderegg) [reply services/director-v2/src/simcore_service_director_v2/api/routes/computations.py:576]: the last push from the train failed obviously

**Pull request merged**